views/js/qtiCreator/widgets/static/text/states/Active.js

```diff
@@ -1,16 +1,19 @@
 define([
+    'services/features',
     'taoQtiItem/qtiCreator/widgets/states/factory',
     'taoQtiItem/qtiCreator/widgets/static/states/Active',
     'taoQtiItem/qtiCreator/editor/ckEditor/htmlEditor',
     'taoQtiItem/qtiCreator/editor/gridEditor/content',
     'taoQtiItem/qtiCreator/widgets/helpers/formElement',
     'tpl!taoQtiItem/qtiCreator/tpl/forms/static/text',
     'taoQtiItem/qtiCreator/widgets/static/helpers/itemScrollingMethods'
-], function (stateFactory, Active, htmlEditor, content, formElement, formTpl, itemScrollingMethods) {
+], function (features, stateFactory, Active, htmlEditor, content, formElement, formTpl, itemScrollingMethods) {
     'use strict';
 
     const wrapperCls = 'custom-text-box';
 
+    const isHiddenPlugin = pluginName =>
+        !features.isVisible(`taoQtiItem/qtiCreator/widgets/static/text/ckeditor/plugins/${pluginName}`);
```

Comment:
suggestion: can we reduce the key size? 

```
item/editor/plugins/taotooltip
```

or even 
```
item/contentEditor/taotooltip
```

Reply:
I think it's better if the key relies on the structure of the code and the place where it's value is used. It's easier for maintainer to understand where to find the code this option is used. It degrades a bit a look of config file and reference doc, though.